samples/FunctionApp/Program.cs

```diff
@@ -21,11 +18,10 @@ static async Task Main(string[] args)
             //<docsnippet_startup>
             var host = new HostBuilder()
                 //<docsnippet_configure_defaults>
-                .ConfigureFunctionsWorkerDefaults(builder =>
+                .ConfigureFunctionsWorkerDefaults((WorkerOptions options) =>
                 {
-                    builder
```

Comment:
AppInsights logic is no longer needed in this sample?

Reply:
This change won't go into the final commit -- was just for testing locally.